### PR TITLE
fix some comments

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: apis/projectcontour/v1/httpproxy.go [hunk 1]
```diff
@@ -1151,7 +1151,7 @@ type LoadBalancerPolicy struct {
 	// `RequestHash` load balancing strategy is chosen. If an element of the
 	// supplied list of hash policies is invalid, it will be ignored. If the
 	// list of hash policies is empty after validation, the load balancing
-	// strategy will fall back the the default `RoundRobin`.
+	// strategy will fall back thedefault `RoundRobin`.
```
Comment (skriss) [inline apis/projectcontour/v1/httpproxy.go:1154]:
```suggestion
	// strategy will fall back to the default `RoundRobin`.
```

**Review by skriss — CHANGES_REQUESTED** [review]
@cuishuang it looks like there are still many typos/grammatical issues in your updates, can you take another pass through this? Thanks.

File: apis/projectcontour/v1/httpproxy.go [hunk 2]
```diff
@@ -1151,7 +1151,7 @@ type LoadBalancerPolicy struct {
 	// `RequestHash` load balancing strategy is chosen. If an element of the
 	// supplied list of hash policies is invalid, it will be ignored. If the
 	// list of hash policies is empty after validation, the load balancing
-	// strategy will fall back the the default `RoundRobin`.
+	// strategy will fall back the  default `RoundRobin`.
```
Comment (sunjayBhatia) [inline apis/projectcontour/v1/httpproxy.go]:
```suggestion
	// strategy will fall back to the default `RoundRobin`.
```

File: cmd/contour/ingressstatus.go [hunk 3]
```diff
@@ -33,7 +33,7 @@ import (
 // updates for HTTPProxy, Ingress and Gateway objects. Actually updating the
 // address in the object status is performed by k8s.StatusAddressUpdater.
 //
-// The theory of operation of the loadBalancerStatusWriter is as follows:
+// the ory of operation of the loadBalancerStatusWriter is as follows:
```
Comment (sunjayBhatia) [inline cmd/contour/ingressstatus.go]: This should not be changed

File: internal/provisioner/model/model.go [hunk 4]
```diff
@@ -552,7 +552,7 @@ type Port struct {
 	// ContainerPort is the port to expose on the Envoy container(s).
 	ContainerPort int32
 	// NodePort is the network port number to expose for the NodePort Service.
-	// If unspecified, a port number will be assigned from the the cluster's
+	// If unspecified, a port number will be assigned from the  cluster's
```
Comment (sunjayBhatia) [inline internal/provisioner/model/model.go]:
```suggestion
	// If unspecified, a port number will be assigned from the cluster's
```

here and other places there should not be an extra space

**Review by sunjayBhatia — CHANGES_REQUESTED** [review]

File: apis/projectcontour/v1/httpproxy.go [hunk 5]
```diff
@@ -1151,7 +1151,7 @@ type LoadBalancerPolicy struct {
 	// `RequestHash` load balancing strategy is chosen. If an element of the
 	// supplied list of hash policies is invalid, it will be ignored. If the
 	// list of hash policies is empty after validation, the load balancing
-	// strategy will fall back the the default `RoundRobin`.
+	// strategy will fall back the default `RoundRobin`.
```
Comment (sunjayBhatia) [inline apis/projectcontour/v1/httpproxy.go:1154]: again, `to the` here.

File: examples/contour/01-crds.yaml [hunk 6]
```diff
@@ -6975,7 +6975,7 @@ spec:
                       type: string
                     targetNamespaces:
                       description: required, the namespaces the authority to reference
-                        the the secret will be delegated to. If TargetNamespaces is
+                        the secret will be delegated to. If TargetNamespaces is
```
Comment (skriss) [inline examples/contour/01-crds.yaml:6978]: You will need to make this change in the `tlscertificatedelegation.go` file in `apis/projectcontour/v1`, and then run `make generate to ensure all of the generated YAML is in sync with it.

Comment (skriss) [reply examples/contour/01-crds.yaml]: Still need to run `make generate` and commit the results..

Comment (skriss) [reply examples/contour/01-crds.yaml]: Hmm, not sure what error you're getting. Is it OK if I push the update to your branch?

Comment (skriss) [reply examples/contour/01-crds.yaml]: Done

**Review by skriss — APPROVED** [review]
LGTM, thanks @cuishuang

**Pull request merged**